src/main/java/de/tum/cit/aet/artemis/plagiarism/web/PlagiarismCaseResource.java

```diff
@@ -270,4 +275,21 @@ public ResponseEntity<PlagiarismCase> getPlagiarismCaseForStudent(@PathVariable
 
         return getPlagiarismCaseResponseEntity(plagiarismCase);
     }
+
+    /**
+     * Informs the instructor about a reply to a plagiarism post by a student.
+     *
+     * @param postId the ID of the post
+     * @return a ResponseEntity with status 200 (OK)
+     */
+    @GetMapping("posts/{postId}/inform-instructor")
+    @EnforceAtLeastStudent
+    public ResponseEntity<Void> informInstructorAboutPostReply(@PathVariable Long postId) {
+        log.debug("GET informInstructorAboutPostReply invoked for post {}", postId);
```

Comment:
There seems to be no validation that the postId is related to a plagiarism case post. With this, students could _theoretically_ trigger a lot of notifications for an instructor.

Reply:
This comment does not seem to be resolved. There is also no check that the student is actually allow to access the post at all